packages/dashboard/src/layout/LayoutUtils.ts

```diff
@@ -240,6 +240,36 @@ class LayoutUtils {
     return null;
   }
 
+  /**
+   * Gets a content item by its ID
+   * @param item Golden layout content item to search for the content item
+   * @param searchId the ID
+   */
+  static getContentItemById(
```

Comment:
I'm surprised this didn't already exist.

Reply:
It looks like we have `PanelManager.getContainerByPanelId` which is close. I'm going to leave this util I think because the closest we can do is the logic from that which gets the stack for the item with config containing the id, then gets the item from the stack

This also lets you get a row or column which wouldn't work with the other logic (since they're not in a stack), but I can't think of a useful reason for that.